EateryModel/Sources/EateryModel/Networking/EateryAPI.swift

```diff
@@ -22,13 +22,9 @@ public struct EateryAPI {
 
     public func eateries() async throws -> [Eatery] {
         let (data, _) = try await URLSession.shared.data(from: url, delegate: nil)
-        let schemaApiResponse = try decoder.decode(Schema.APIResponse.self, from: data)
+        let schemaApiResponse = try decoder.decode([Schema.Eatery].self, from: data)
 
-        if let error = schemaApiResponse.error {
-            throw EateryAPIError.apiResponseError(error)
-        }
-
-        return schemaApiResponse.data.map(SchemaToModel.convert)
+        return schemaApiResponse.map(SchemaToModel.convert)
```

Comment:
Is there a reason why the error handling here was deleted?

Reply:
you're totally right i shouldn't have deleted it. will add it back and request review again